src/openforms/config/templates/admin/config/overview.html

```diff
@@ -29,7 +29,7 @@ <h1>{% trans 'Configuration' %}</h1>
                         {% for entry in section.entries %}
                             <tr>
                                 <td><strong>{{ entry.name }}</strong></td>
-                                <td>{{ entry.status_icon }}{% if entry.status_message %} {{ entry.status_message|linebreaksbr }}{% endif %}</td>
+                                <td {% if entry.error %}title="{{ entry.error }}"{% endif %}>{{ entry.status_icon }}{% if entry.status_message %} {{ entry.status_message|linebreaksbr }}{% endif %}</td>
```

Comment:
Requirement:

> Refactor the the view to show only the error message: "Failed to validate the configuration." in case of an error. In the alt/title attribute, you can put the whole error. This should clean up the view.

The literal text "Failed to validate the configuration." may be missing if there was no exception raised.

```suggestion
                                <td {% if entry.error %}title="{{ entry.error }}"{% endif %}>{{ entry.status_icon }}{% if entry.status == False %}{% trans "Failed to validate the configuration." %}{% endif %}</td>
```

We don't need `status_message` at all anymore I think.